print("exception handling")
print("exception handling")

try:
    print(x)
except:
    print("client side error")

print("exception handling successful")
print("exception handling successful")